src/Symfony/Component/Serializer/Annotation/Context.php

```diff
@@ -33,14 +33,14 @@ public function __construct(
         string|array $groups = [],
```

Comment:
The namespace is still `...Component\Serializer\Annotation`. Worth deprecating and move to `Attribute`?